profiler/quickstart/main.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""An example of starting https://cloud.google.com/profiler."""
+
+# [START profiler_python_quickstart]
+import googlecloudprofiler
+
+
+def main():
+    # Profiler initialization. It starts a daemon thread which continuously
+    # collects and uploads profiles. Best done as early as possible.
+    try:
+        googlecloudprofiler.start(
+            service='hello-profiler',
+            service_version='1.0.1',
+            verbose=3,
```

Comment:
3 is a magic number. Does the client library provide any constants which may be used instead? Otherwise create a constant for the sample.

Reply:
Done.

Reply:
I just added a comment. The client code doesn't provide constants for logging levels. This sample will serve as a code snippet of calling the start function. I'm a little hesitate to create a constant, which may confuse the reader.

Reply:
Is there a FR to add such constants to the client? I think such constants would be useful.

Reply:
Created a FR. Though I'm not sure if we definitely want to do that change, the number is consistent with the profiling agents in other languages.